var/spack/repos/builtin/packages/py-onnxruntime/package.py

```diff
@@ -31,6 +57,7 @@ class PyOnnxruntime(CMakePackage, PythonExtension):
     depends_on("cxx", type="build")  # generated
 
     variant("cuda", default=False, description="Build with CUDA support")
+    variant("rocm", default=False, description="Build with ROCm support")
```

Comment:
Why don't we use the mixin `ROCmPackage` here? (Same of course applies to CUDA, but just do the "right thing" for ROCm for now.)

Reply:
Done!